Core.ApplicationServices/GDPR/DataProcessingRegistrationApplicationService.cs

```diff
@@ -503,5 +504,20 @@ public Result<(DataProcessingRegistration registration, IEnumerable<DataProcessi
                 return registration;
             });
         }
+
+        public IQueryable<DataProcessingRegistration> GetDataProcessingRegistrationsByOrganization(int orgId, params IDomainQuery<DataProcessingRegistration>[] conditions)
```

Comment:
Organization behøves ikke at være påkrævet og hvis den gør, så skal du nok lige huske access control her :-)

Reply:
Og hvis org id fjernes skal du stadig huske at filtrere base query ift. enten FULL cross org access eller alle brugerens orgs (se i IT-system usage service i min branch)

Reply:
husk også en unit test